camel/generators.py

```diff
@@ -29,13 +29,16 @@ class SystemMessageGenerator:
         sys_msg_meta_dict_keys (Optional[Set[str]], optional): The set of keys
             of the meta dictionary used to fill the prompts.
             (default: :obj:`None`)
+        with_role_description (bool, optional): Whether to include the role
```

Comment:
Test SystemMessageGenerator in both modes: without and with the role description.